packages/core/src/lib/context/api.ts

```diff
@@ -0,0 +1,102 @@
+import { GraphQLSchema } from 'graphql';
+import type { InitialisedList } from '../core/types-for-lists';
+import { KeystoneContext } from '../../types';
+import { executeGraphQLFieldToRootVal } from './executeGraphQLFieldToRootVal';
+import { executeGraphQLFieldWithSelection } from './executeGraphQLFieldWithSelection';
+
+export function getQueryFactory(list: InitialisedList, schema: GraphQLSchema) {
+  function f(operation: 'query' | 'mutation', field: string) {
```

Comment:
Could the `field` `fieldName` naming be standardised? - `Field` in my head always goes to a Keystone Field instance, where this is referring to the GrpahQL field which is the query/mutation name